docs/quickstart.rst

```diff
@@ -0,0 +1,69 @@
+Quickstart guide
```

Comment:
I wonder if there is a way to pull this content from the `README.md`?

Reply:
We could add a step to `build_docs.py` to do that. It would make things easier if we converted it from markdown to ReST.